site/en/r2/tutorials/load_data/numpy.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>Why not use tf.keras.dataset.mnist.load_data()?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/606/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
I think the point is to explicitly show loading a .npz file

Reply:
Not, as I understand it, `.npz` specifically - but in-scope NumPy arrays.